src/vs/workbench/contrib/issue/browser/issueQuickAccess.ts

```diff
@@ -27,21 +26,17 @@ export class IssueQuickAccess extends PickerQuickAccessProvider<IPickerQuickAcce
 		@IContextKeyService private readonly contextKeyService: IContextKeyService,
 		@ICommandService private readonly commandService: ICommandService,
 		@IExtensionService private readonly extensionService: IExtensionService,
-		@IConfigurationService private readonly configurationService: IConfigurationService,
 		@IProductService private readonly productService: IProductService
 	) {
 		super(IssueQuickAccess.PREFIX, { canAcceptInBackground: true });
 	}
 
 	protected override _getPicks(filter: string): Picks<IPickerQuickAccessItem> | FastAndSlowPicks<IPickerQuickAccessItem> | Promise<Picks<IPickerQuickAccessItem> | FastAndSlowPicks<IPickerQuickAccessItem>> | null {
-		if (!this.configurationService.getValue<boolean>('extensions.experimental.issueQuickAccess')) {
-			return null;
-		}
-		const issuePicks: Array<IPickerQuickAccessItem | IQuickPickSeparator> = [];
-		const extensionIdSet: Set<string> = new Set();
+		const issuePicks = new Array<IPickerQuickAccessItem | IQuickPickSeparator>();
+		const extensionIdSet = new Set<string>();
 
 		// add regular open issue reporter button
-		const productLabel = this.productService.nameLong === 'stable' ? localize("workbench.action.openIssueReporter", "Visual Studio Code") : localize("workbench.action.openIssueReporterInsiders", "Visual Studio Code: Insiders");
+		const productLabel = localize("workbench.action.openIssueReporter.productName", "{0}", this.productService.nameLong);
```

Comment:
bruh

```ts
const productLabel = this.productService.nameLong;
```

Reply:
i wanted to localize in case HAHA 😭

Reply:
Remember that what gets sent to translators is the first and second parameter...so the literal string: `"{0}"` is what needs to be translated. That's not translatable.